pkg/sql/compile/cnserviceDriver.go

```diff
@@ -243,6 +246,9 @@ func sendBackBatchToCnClient(ctx context.Context, b *batch.Batch, messageId uint
 		m := mHelper.acquirer().(*pipeline.Message)
 		m.Id = messageId
 		m.Data = encodeData
+		m.Checksum = crc32.ChecksumIEEE(m.Data)
+		m.Sequence = mHelper.sequence
+		mHelper.sequence++
```

Comment:
Here is a concurrency scenario where you need to consider concurrency security, as well as performance